content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
 <Section>
 <Text>
 
+### Get Customer
+
+Returns the info about the Customer with given id.
+
+#### Specifics
+
+|                        |                                               |
+| ---------------------- | --------------------------------------------- |
+| **Action**             | `get_customer`                                |
+| **Required scopes**    | `customers:ro`                                |
+| **RTM API equivalent** | [`get_customer`](rtm-reference/#get-customer) |
+| **Push message**       | -                                             |
+
+#### Request
+
+| Parameter     | Required | Data type | Notes |
+| ------------- | -------- | --------- | ----- |
+| `customer_id` | Yes      | string    |       |
+
+#### Response
+
+|              |                                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`         | [Customer's](#customer) id.                                                                                                            |
+| `type`       | `customer`                                                                                                                             |
```

Comment:
I suggest we link to the **customer object** here and remove the links in other fields (name, email, avatar, etc.) I know this pattern is copied from Get Customer in Customer API, but it seems like 6 links pointing to the same place is too much.